test/test_backend.py

```diff
@@ -19,10 +21,48 @@ def setup(self):
         """always runs first"""
         pass
 
-    def tearDown(self) -> None:
+    def tearDown(self):
         """always runs last"""
         pass
 
+    def test_hierarchify_list_of_parent_kids(self):
```

Comment:
@Sigfried: You might want to check out this unit test. The second test case made it a lot easier for me to tell that it was working correctly.

Reply:
@joeflack4: Also not working for me. I can't figure out how to run the tests in the debugger, but from command line I get:
[junk.txt](https://github.com/jhu-bids/TermHub/files/10268087/junk.txt)

Doesn't seem like that test is running at all

Reply:
Ok...I couldn't run the tests in the usual way, but just ran them manually in the debugger (line by line) and see that they work. Not sure why I get something different running the endpoint